### DISCUSSION TITLE
Linux latency, extended

### DISCUSSION THREAD

**Discussion by DarkGradient** [ppy/osu#31209] [Q&A] [answered]
If it's not too much trouble, can you explain to me why in Linux, when playing osu! there is a **delay** of **1.2ms**?
What is it **related** to?
I just don't understand **why** it's like that.

_Srry, if i wasting your time 😣_

**Comment by peppy** [comment, marked as answer]
1.2ms is how often we're computing game logic. This is a good rate and you should not be concerned.

If it bothers you, please hide the display.
